ExtensionBundle.Tests/TestData/any_any_extensions.deps.json

```diff
@@ -931,79 +931,81 @@
           }
         }
       },
-      "Microsoft.Azure.DurableTask.ApplicationInsights/0.1.6": {
+      "Microsoft.Azure.DurableTask.ApplicationInsights/0.2.0": {
         "dependencies": {
           "Microsoft.ApplicationInsights": "2.21.0",
-          "Microsoft.Azure.DurableTask.Core": "2.17.1",
+          "Microsoft.Azure.DurableTask.Core": "3.0.0",
           "Microsoft.Bcl.AsyncInterfaces": "6.0.0"
         },
         "runtime": {
           "lib/netstandard2.0/DurableTask.ApplicationInsights.dll": {
-            "assemblyVersion": "0.1.0.0",
-            "fileVersion": "0.1.6.6114"
+            "assemblyVersion": "0.2.0.0",
+            "fileVersion": "0.2.0.54820"
           }
         }
       },
-      "Microsoft.Azure.DurableTask.AzureStorage/1.17.3": {
+      "Microsoft.Azure.DurableTask.AzureStorage/2.0.1": {
         "dependencies": {
-          "Azure.Data.Tables": "12.8.0",
-          "Microsoft.Azure.DurableTask.Core": "2.17.1",
-          "Microsoft.Extensions.Logging.Abstractions": "6.0.0",
-          "WindowsAzure.Storage": "9.3.1"
+          "Azure.Core":"1.41.0",
+          "Azure.Data.Tables": "12.9.1",
+          "Azure.Storage.Blobs": "12.20.0",
+          "Azure.Storage.Queues":"12.18.0",
+          "Microsoft.Azure.DurableTask.Core": "3.0.0",
+          "Microsoft.Extensions.Logging.Abstractions": "6.0.0"

```

Comment:
Should this be 6.0.1 since DurableTask.Core depends on >= 6.0.1? https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Core